config/pipeline.yaml

```diff
@@ -812,19 +815,42 @@ build_variants:
     gcc-10:
       build_environment: gcc-10
       architectures:
-        x86_64:
+        x86_64: &x86_64_defconfig
           base_defconfig: 'x86_64_defconfig'
           filters:
             - regex: { defconfig: 'x86_64_defconfig' }
-        arm64:
+        arm64: &arm64_defconfig
           base_defconfig: 'defconfig'
           filters:
             - regex: { defconfig: 'defconfig' }
-        arm:
+        arm: &arm_defconfig
           base_defconfig: 'multi_v7_defconfig'
           filters:
             - regex: { defconfig: 'multi_v7_defconfig' }
 
+preempt_rt_variants: &preempt_rt_variants
+  gcc-10:
+    build_environment: gcc-10
+    fragments: [preempt_rt]
+    architectures:
+      arm:
+        <<: *arm_defconfig
+        extra_configs:
+          - 'multi_v7_defconfig+preempt_rt'
+      arm64:
+        <<: *arm64_defconfig
+        fragments:
+          - 'arm64-chromebook'
+        extra_configs:
+          - 'defconfig+preempt_rt'
+          - 'defconfig+preempt_rt+arm64-chromebook'
+      x86_64:
+        <<: *x86_64_defconfig
+        fragments:
+          - 'x86-board'
+        extra_configs:
+          - 'x86_64_defconfig+preempt_rt'
+          - 'x86_64_defconfig+preempt_rt+x86-board'
```

Comment:
I am not sure if this is causing issues but when I compared these variants with `build_variants`, it looks like `variants` field is missing from the definition.
See https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml#L832

Reply:
I've added variants and run offline. After changes other than fragments, changes are working fine.